tianshou/evaluation/rliable_evaluation_hl.py

```diff
@@ -216,3 +320,30 @@ def eval_results(
             plt.show(block=False)
 
         return fig_iqm, ax_iqm, fig_profile, ax_profile
+
+
+def load_and_eval_experiments(
+    log_dir: str,
+    show_plots: bool = True,
+    save_plots: bool = True,
+    scope: DataScope = DataScope.TEST,
+    max_env_step: int | None = None,
+) -> RLiableExperimentResult:
+    """Evaluate the experiments in the given log directory using the rliable API and return the loaded results object.
+
+    If neither `show_plots` nor `save_plots` is set to `True`, this is equivalent to just loading the results from disk.
+
+    :param log_dir: The directory containing the experiment results.
+    :param show_plots: whether to display plots.
+    :param save_plots: whether to save plots to the `log_dir`.
+    :param scope: The scope of the evaluation, either 'TEST' or 'TRAIN'.
+    :param max_env_step: The maximum number of environment steps to consider. If None, all data is considered.
+            Note: if the experiments have different numbers of steps, the minimum number is used.
+    """
+    rliable_result = RLiableExperimentResult.load_from_disk(log_dir, max_env_step=max_env_step)
+    if scope == "both":
```

Comment:
scope `both` currently doesn't exist in `DataScope`

Reply:
Ah, it was supposed to be a union type, good catch! A shame mypy doesn't see it

Reply:
Fixed